NOTICE-executable

```diff
@@ -0,0 +1,637 @@
+Embulk
+Copyright 2014-2023 Sadayuki Furuhashi, and the Embulk project.
+
+This product includes software developed by
+Sadayuki Furuhashi, and the Embulk project (https://www.embulk.org/).
+
+
+
+The executable distribution of this product includes
+the API for SLF4J (The Simple Logging Facade for Java), developed by
+QOS.ch Sarl (Switzerland), licensed under
+the MIT License (http://www.opensource.org/licenses/mit-license.php).
+
+* Copyright (c) 2004-2022 QOS.ch Sarl (Switzerland)
+
+
+
+The executable distribution of this product includes
+Logback, developed by QOS.ch, dual-licensed under
+the Eclipse Public License v1.0, or the GNU Lesser General Public License version 2.1.
+
+* Copyright (C) 1999-2015, QOS.ch. All rights reserved.
+
+Note that this product adopts the Eclipse Public License v1.0 for Logback.
+Refer to the license of Logback at META-INF/LICENSE-logback.
+
+
+
+The executable distribution of this product includes
+MessagePack for Java, licensed under
+the Apache License, Version 2.0.
+
+* The NOTICE file of MessagePack for Java is embedded in this NOTICE below.
+
+
+
+The executable distribution of this product includes
+Jackson JSON processor, developed by
+Tatu Saloranta, tatu.saloranta@iki.fi, licensed under
+the Apache License, Version 2.0.
+
+* Copyright (c) 2007- Tatu Saloranta, tatu.saloranta@iki.fi
+* The NOTICE file of Jackson JSON processor is embedded in this NOTICE below.
+
+
+
+The executable distribution of this product includes
+Slice, developed by the Airlift project, licensed under
+the Apache License, Version 2.0.
+
+
+
+The executable distribution of this product includes
+Netty/Common and Netty/Buffer, developed by
+The Netty Project (http://netty.io/), licensed under
+the Apache License, Version 2.0.
+
+* Copyright 2011 The Netty Project
+* The NOTICE.txt file of Netty is embedded in this NOTICE below.
+
+Note that Netty/Common contains modified portions of some other products.
+
+Netty/Common contains the extensions to Java Collections Framework which has
+been derived from the works by JSR-166 EG, Doug Lea, and Jason T. Greene.
+Refer to the license at META-INF/LICENSE-jsr166, instead of license/LICENSE.jsr166y.txt.
```

Comment:
META-INF/LICENSE-jsr166y ?

Reply:
Fixed it in: f5ecc1b65e0776dc982ee4ed323f18cee8d822f8